index.html

```diff
@@ -5,9 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 </head>
 <body>
-    <header></header>
-    <section></section>
+    <header>
+        <div class="status-bar"></div>
+        <div class="toolbar">
+            <div class="logo-placeholder">
+                <p class="name white-color">Thomas Obuya</p>
+            </div>
+            <div class="menu-placeholder">
+                <button class="menu">
+                    <img src="menu.svg" alt="Menu">
+                </button>
+            </div>
+        </div>
+    </header>
+    <section class="headline">
+        <div class="headline-bio">
+            <p class="primary-text">Hey There. I'm Thomas</p>
+            <p class="supporting-text">I'am a Software Developer</p>
+            <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class="social-media">
+            <div class="frame">
+                <button>
+                    <img class="github" src="Icon -GitHub.svg" alt="Github">
+                </button>
+            </div>
+            <div class="frame">
+                <button>
+                    <img src="linkedin.svg" alt="LinkedIn">
+                </button>
+            </div>
+            <div class="frame">
+                <button>
+                    <img src="angelist 1.svg" alt="Chill">
+                </button>
+            </div>
+            <div class="frame">
+                <button>
+                    <img src="twitter.svg" alt="Twitter">
+                </button>
+            </div>
+            <div class="frame">
+                <button>
+                    <img src="Vector.svg" alt="Vector">
```

Comment:
- [ ] I think it will be better if the list of your social icons uses a `ul` tag to group them, and `li` per icon; with each icon wrapped in a `<a>` tag. This is so because the icons are a list (`<li>`) and for semantic purposes, it is best to wrap them in a list (`<ul>`). For it to be clickable so that it can take you to the original social media page, it needs to be wrapped in an `<a>` tag 🤓 

e.g.

```
<ul>
    <li>
        <a href="#" >
              <img src="twitter.svg" alt="Twitter">
         </a>
    </li>
    <li>
        <a href="#" >
              <img src="github.svg" alt="github">
         </a>
    </li>
</ul>
```